internal/cli/release_list.go

```diff
@@ -0,0 +1,391 @@
+			if t, err := ptypes.Timestamp(b.Status.StartTime); err == nil {
+				startTime = humanize.Time(t)
+			}
+			if t, err := ptypes.Timestamp(b.Status.CompleteTime); err == nil {
+				completeTime = humanize.Time(t)
+			}
+
+			// Add status report information if we have any
+			statusReportComplete := "n/a"
+			if releaseBundle.LatestStatusReport != nil {
+				statusReport := releaseBundle.LatestStatusReport
+				switch statusReport.Health.HealthStatus {
+				case "READY":
+					statusReportComplete = "✔"
+				case "ALIVE":
+					statusReportComplete = "✔"
+				case "DOWN":
+					statusReportComplete = "✖"
+				case "PARTIAL":
+					statusReportComplete = "●"
+				case "UNKNOWN":
+					statusReportComplete = "?"
+				}
+
+				if t, err := ptypes.Timestamp(statusReport.GeneratedTime); err == nil {
+					statusReportComplete = fmt.Sprintf("%s - %s", statusReportComplete, humanize.Time(t))
+				}
+			}
+
+			var (
+				extraDetails []string
+				details      []string
+			)
+
+			if user, ok := b.Labels["common/user"]; ok {
+				details = append(details, "user:"+user)
+			} else if b.Preload.Build != nil {
+				build := b.Preload.Build
+				// labels have been set, safe to use them
+
+				if user, ok := build.Labels["common/user"]; ok {
+					details = append(details, "build-user:"+user)
+				}
+				if bp, ok := build.Labels["common/languages"]; ok {
+					details = append(details, niceLanguages(bp))
+				}
+
+				if img, ok := build.Labels["common/image-id"]; ok {
+					img = shortImg(img)
+
+					details = append(details, "image:"+img)
+				}
+			}
+
+			if b.Preload.Artifact != nil {
+				artDetails := fmt.Sprintf("artifact:%s", c.flagId.FormatId(b.Preload.Artifact.Sequence, b.Preload.Artifact.Id))
+				if len(details) == 0 {
+					details = append(details, artDetails)
+				} else if c.flagVerbose && b.Preload.Build != nil {
+					details = append(details,
+						artDetails,
+						fmt.Sprintf("build:%s", c.flagId.FormatId(b.Preload.Build.Sequence, b.Preload.Build.Id)))
+				}
+			}
+
+			if c.flagVerbose {
+				for k, val := range b.Labels {
+					if strings.HasPrefix(k, "waypoint/") {
+						continue
+					}
+
+					if len(val) > 30 {
+						val = val[:30] + "..."
+					}
+
+					extraDetails = append(extraDetails, fmt.Sprintf("Release.%s:%s", k, val))
+				}
+				sort.Strings(extraDetails)
+			}
+
+			sort.Strings(details)
+			var firstDetails string
+			if len(details) > 0 {
+				firstDetails = details[0]
+			}
+
+			var columns []string
+
+			columns = []string{
+				status,
+				c.flagId.FormatId(b.Sequence, b.Id),
+				c.flagId.FormatId(b.Preload.Deployment.Sequence, b.Id),
+				b.Component.Name,
+				firstDetails,
+				startTime,
+				completeTime,
+				statusReportComplete,
+			}
+
+			if c.flagUrl {
+				url := "n/a"
+				if b.Url != "" {
+					url = b.Url
+				} else if releaseBundle.Release.Url != "" {
+					url = releaseBundle.Release.Url
+				}
+				columns = append(columns, url)
+			}
+
+			if !b.Unimplemented {
```

Comment:
We should add a brief doc comment note here saying we won't actually list any releases that didn't actually happen.